### DISCUSSION TITLE
Frigate+: make it possible to mask off area(s) in image not relevant to training

### DISCUSSION THREAD

**Discussion by ProsjektX** [blakeblackshear/frigate#20042] [Ideas]
Just wonder if it would have been useful to be able to mask off a section (or sections) of an image which shall not be used in training? Both to prevent over-training on regular objects that frequently appear in specific areas, and to make it easier to tag objects of primary interest to train on.

**Comment by NickM-27** [comment]
> Both to prevent over-training on regular objects that frequently appear in specific areas

If there are objects in the area then they need to be labeled, or you'll be teaching the model that those are not true objects.

The images are only used for fine-tuning so overfitting is very unlikely.

**Reply by ProsjektX** [reply]
> > Both to prevent over-training on regular objects that frequently appear in specific areas
> 
> If there are objects in the area then they need to be labeled, or you'll be teaching the model that those are not true objects.

yeah, that's why I was talking about a mask so any objects behind the mask would not be seen when training.
> 
> The images are only used for fine-tuning so overfitting is very unlikely.

ok, if that's the case then maybe not an issue. Just thinking I keep labeling the same objects over and over, when the actual objects of interest are much less frequent, and what I should focus on.

**Reply by NickM-27** [reply]
My neighbor had the same car, broken down, parked for the last two years. So it was included in 1600 images from that camera, it was only just fixed up and moved and it does not have any signs of overfitting.

The suggestions feature should also label most objects when you submit the image anyway
